ci-operator/step-registry/openshift-qe/rosa-upgrade-postcheck/openshift-qe-rosa-upgrade-postcheck-workflow.yaml

```diff
@@ -0,0 +1,8 @@
+
+workflow:
```

Comment:
since Rosa uses a worfklow to create the cluster (and you can only use 1 workflow per test item in the config), I dont think we could ever use this file. Think we can delete this

Reply:
+1 we should delete this.

Reply:
Ok， already delete in new code